prdoc/pr_5915.prdoc

```diff
@@ -0,0 +1,17 @@
+# Schema: Polkadot SDK PRDoc Schema (prdoc) v1.0.0
+# See doc at https://raw.githubusercontent.com/paritytech/polkadot-sdk/master/prdoc/schema_user.json
+
+title: Omni-Node renamings
+
+doc:
+  - audience: Node Dev
+    description: |
+      This PR renames the `polkadot-parachain-lib` crate to `polkadot-omni-node-lib`.
```

Comment:
We could mention the introduction of `polkadot-omni-node` binary.

Reply:
Good point ! Done.